docs/ui/components/Home/sections/DiscoverMore.tsx

```diff
@@ -45,13 +45,13 @@ export function DiscoverMore() {
           <OfficeHoursImage />
           <RawH3 className="!text-palette-yellow11 !font-bold">Join us for Office Hours</RawH3>
           <P className="!text-palette-yellow11 !text-xs max-w-[28ch]">
-            Get answers to your questions and get advice from the Expo team.
+            Ask the Expo Team questions on our Discord stage.
```

Comment:
```suggestion
            Ask the Expo Team questions during our weekly office hours on Discord.
```
^^ something like that to highlight that it's a recurring weekly thing. Else it might give the false impression that you can ask any time an expect someone from Expo to reply.

Reply:
it's not every week